### ISSUE TITLE
Multiple Instance Learning performance

### ISSUE THREAD

**Issue by gsdndeer** [Project-MONAI/tutorials#1154]
Hi all,

I am using the code from the tutorial for Multiple Instance Learning from here:  [https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py](https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py)

I used my own dataset, which are also prostate histology WSIs, but the results are overfitting and different from the ones in the readme. Is there any way to deal with this? Thank you.

Below are my train and validation loss curves:

<img width="1246" alt="Screen Shot 2023-01-09 at 7 14 36 PM" src="https://user-images.githubusercontent.com/51049984/211649519-16944089-609a-48cf-b2fc-e40ec9dbbb1a.png">

**Comment by KumoLiu** [comment]
Hi @gsdndeer, there is a typo in the normalization during preprocessing, not sure whether it's the reason. Could you please invert `a_max` and `a_min` values and try training again?
https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py#L296
https://github.com/Project-MONAI/tutorials/blob/5a497332b2b9df201d59e60f9f1062a11babc144/pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py#L313

Thanks!
